src/components/search/RecommendedCourseCard.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Truncate from 'react-truncate';
+import { Link } from 'react-router-dom';
+import Skeleton from 'react-loading-skeleton';
+import { Card } from '@edx/paragon';
+
+/**
+ * A component to be be rendered in carousel as carouselItem. The component should be able to render carousel item
+ * loading state if isLoading prop is passed as true.
+ * @param props {classes} string (optional) - Custom classes to be applied to carouselItem based on number of items
+ * @param props {isLoading} Bool - If isLoading is true, the component will render loading state else actual state.
+ * @param props {itemData} Object - Single item data from the carouselData list. Passed from carousel.
+ */
+const RecommendedCourseCard = (props) => {
+  const {
+    itemData: course,
+    isLoading,
+    classes,
+  } = props;
+  const loadingCard = () => (
+    <Card>
+      <Card.ImageCap
+        as={Skeleton}
+        duration={0}
+      />
+
```

Comment:
Is adding extra line between statements being followed in learner portal code? If yes, then leave it for the sake of convention otherwise lets remove these extra lines.

Reply:
They did the same